src/a-d/Dema/Dema.BufferList.cs

```diff
@@ -3,7 +3,7 @@ namespace Skender.Stock.Indicators;
 /// <summary>
 /// Double Exponential Moving Average (DEMA) from incremental reusable values.
 /// </summary>
-public class DemaList : List<DemaResult>, IDema, IBufferList, IBufferReusable
+public class DemaList : BufferList<DemaResult>, IDema, IBufferList, IBufferReusable
```

Comment:
Per your earlier comment, is this better?  I think it already inherits `IBufferList`

```suggestion
public class DemaList : BufferList<DemaResult>, IDema, IBufferReusable
```